web/client/components/theme/Theme.jsx

```diff
@@ -12,8 +12,13 @@ const ConfigUtils = require('../../utils/ConfigUtils');
 
 const reducePropsToState = (props) => {
     const innermostProps = props[props.length - 1];
-    if (innermostProps) {
+    /**
+     * version is taken from the build.hs which inject something like SNAPSHOT-lastSHACommit
```

Comment:
This is not relevant here, the component is not aware of an external world existing, it lives on its own desert island.
Remove this comment.